quickwit/quickwit-proto/protos/quickwit/metastore.proto

```diff
@@ -41,6 +41,8 @@ enum SourceType {
   SOURCE_TYPE_PULSAR = 9;
   SOURCE_TYPE_VEC = 10;
   SOURCE_TYPE_VOID = 11;
+  /// Amazon SQS
+  SOURCE_TYPE_SQS = 12;
```

Comment:
This is super confusing...

Is it really SQS? Is it the Queue source? There is pubsub in this list too.
Do we have anything to hint that one is holding message, and hte other is holding urls to files?

Reply:
answered in https://github.com/quickwit-oss/quickwit/pull/5148#discussion_r1662158060